lib/api/src/modules/settings.ts

```diff
@@ -0,0 +1,41 @@
+import { ModuleFn } from '../index';
+
+export interface SubAPI {
+  changeSettingsTab: (tab: string) => void;
+  closeSettings: () => void;
+  navigateToSettingsPage: (path: string) => Promise<void>;
+}
+
+export const init: ModuleFn = ({ store, navigate, fullAPI }) => {
+  const api: SubAPI = {
+    closeSettings: () => {
+      const {
+        settings: { lastTrackedStoryId },
+      } = store.getState();
+
+      if (lastTrackedStoryId) {
+        fullAPI.selectStory(lastTrackedStoryId);
+      } else {
+        navigate('/story/*');
+      }
+    },
+    changeSettingsTab: (tab: string) => {
+      navigate(`/settings/${tab}`);
+    },
+    navigateToSettingsPage: async (path) => {
+      const { settings, storyId } = store.getState();
+
+      await store.setState({
+        settings: { ...settings, lastTrackedStoryId: storyId },
+      });
+
+      navigate(path);
```

Comment:
I guess this means we decided we'd use URLs for settings after all?

Reply:
Yeah. I mean -- I'm quite neutral about it. I figured if the experience would work & be linkable as well, that it wouldn't hurt to keep the URLs around. Any thoughts on that?

Reply:
Sure, I’m not wedded to either approach.